From f2caec9de5c14d047e4a6644653ccfa3835639f0 Mon Sep 17 00:00:00 2001
From: Shunhua Lan <lsh@rock-chips.com>
Date: Tue, 7 Nov 2023 16:35:12 +0800
Subject: [PATCH 196/232] arm64: dts: rockchip:
 rk3568-evb-rk628-rgb2hdmi-ddr4-v10: enable rk628 hdmitx sound card

Signed-off-by: Shunhua Lan <lsh@rock-chips.com>
Change-Id: I75393d8ee20fef8c95929a5674564452e0c5462d
Signed-off-by: Guochun Huang <hero.huang@rock-chips.com>
---
 .../dts/rockchip/rk3568-evb-rk628-rgb2hdmi-ddr4-v10.dts   | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3568-evb-rk628-rgb2hdmi-ddr4-v10.dts b/arch/arm64/boot/dts/rockchip/rk3568-evb-rk628-rgb2hdmi-ddr4-v10.dts
index 5c0f4886a69080..83b62f996affa7 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568-evb-rk628-rgb2hdmi-ddr4-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3568-evb-rk628-rgb2hdmi-ddr4-v10.dts
@@ -38,6 +38,10 @@
 	status = "disabled";
 };
 
+&i2s2_2ch {
+	status = "okay";
+};
+
 &route_rgb {
 	status = "disabled";
 };
@@ -59,3 +63,7 @@
 		};
 	};
 };
+
+&rk628_sound {
+	status = "okay";
+};
-- 
2.17.1

